### PR TITLE
2013 drupal jen1

### PR DESCRIPTION
**Pull request by jenlampton**
PR for https://github.com/backdrop/backdrop-issues/issues/2013

Change record ready: https://api.backdropcms.org/node/44961

### CODE REVIEW

File: core/includes/common.inc [hunk 1]
```diff
@@ -2910,6 +2910,15 @@ function backdrop_deliver_html_page($page_callback_result) {
   global $language;
   backdrop_add_http_header('Content-Language', $language->langcode);
 
+  // By default, do not allow the site to be rendered in an iframe on another
+  // domain, but provide a variable to override this. If the code running for
+  // this page request already set the X-Frame-Options header earlier, don't
+  // overwrite it here.
+  $frame_options = config_get('system.core', 'x_frame_options');
+  if ($frame_options && is_null(drupal_get_http_header('X-Frame-Options'))) {
```
Comment (klonos) [inline core/includes/common.inc]: https://api.backdropcms.org/api/backdrop/core!includes!bootstrap.inc/function/backdrop_get_http_header/1

File: core/modules/system/system.install [hunk 2]
```diff
@@ -2955,6 +2955,15 @@ function system_update_1062() {
 }
 
 /**
+ * Remove configuration settings that were never used.
```
Comment (klonos) [inline core/modules/system/system.install]: I don't think that this description docblock matches what this update hook does.

Comment (jenlampton) [reply core/modules/system/system.install]: It's accurate. that function deleted the user timezone settings from the date config file.

Comment (quicksketch) [reply core/modules/system/system.install]: That is what `system_update_1062()` did. We need a description for the new update, `system_update_1063()`.

**Review by quicksketch — CHANGES_REQUESTED** [review]

**Comment by jenlampton** [comment]
Ah, I see...  sorry I missed that the first time. Fixed and rebased.

**Review by quicksketch — APPROVED** [review]

**Comment by quicksketch** [comment]
Thanks! Merged into 1.x only, as this introduces a minor API difference. I updated the change record to reflect the version being 1.9.0.

**Pull request merged**